Module.Help.Tests.ps1

```diff
@@ -0,0 +1,117 @@
+﻿<#	
+	.NOTES
+		===========================================================================
```

Comment:
As I much as I love Sapien, June, we probably can't ship this block of metadata with ScriptAnalyzer. Do you mind removing it from your branch?